site/public/css/colors.css

```diff
@@ -116,6 +116,10 @@
     --standard-pastel-blue: #d9edf7;
     --standard-light-blue: #99ccff;
     --standard-light-seafoam: #a4dad5;
+    --standard-grey: #808080;
```

Comment:
If any of these colors (or close-enough approximations) exist already in this file, use those instead of adding new ones. For example, we already have various oranges, yellows, & blacks.